distribution/docs/src/main/resources/content/_configuring/uploading-and-editing.adoc

```diff
@@ -0,0 +1,35 @@
+:title: Configuring User Access to Ingest and Metadata for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 02
+:summary: Configuring user ability to upload products and edit existing metadata in ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} lets the administrator control user access to ingest and metadata. The administrator
```

Comment:
✏️ `The administrator can control user access to ingest and metadata in ${catalog-ui}.`